ipa-core/src/test_fixture/input/sharing.rs

```diff
@@ -69,6 +72,47 @@ where
     }
 }
 
+impl<BK, V> IntoShares<HybridReport<BK, V>> for TestRawDataRecord
```

Comment:
with respect to the comment above about using `TestHybridRecord`, you should be able to essentially reuse this with some small tweaks!